Documentation/ceph-upgrade.md

```diff
@@ -56,6 +56,39 @@ released, the process of updating from v1.2.0 is as simple as running the follow
 kubectl -n rook-ceph set image deploy/rook-ceph-operator rook-ceph-operator=rook/ceph:v1.2.2
 ```
 
+If you want to enable the CSI v2.0 driver, additional steps are needed in the
+latest Rook patch release. There are some new features in 2.0 such as
+supporting resizing volumes and support for a new csi attacher. If you don't
+need these features now, you can skip this step for the patch upgrade.
+
+## Enabling the CSI v2.0 driver
+
+To enable the CSI v2.0 driver, you will need to apply updated RBAC settings:
+
+```sh
+kubectl apply -f upgrade-from-v1.2-apply.yaml
```

Comment:
How about if you rename this file to `enable-csi-2.0-rbac.yaml`? Then it's more clear that this purpose is just for csi.

Reply:
@travisn this filename is carried from master branch, if we have to change it , it's good to first change it in master and backport it. Any suggestion?

Reply:
In this case I would actually recommend to just add another commit to the PR to rename it for the release-1.2 branch. In master we will actually want the upgrade guide to include this as an "upgrade-from-1.2" rbac manifest.

Reply:
updated to a new file name